src/web-auth/transaction-manager.js

```diff
@@ -11,20 +11,21 @@ function TransactionManager(options) {
 
 TransactionManager.prototype.process = function(options) {
   var transaction;
+  var responseType = options.responseType || '';
 
-  if (options.responseType.indexOf('code') !== -1) {
+  if (responseType.indexOf('code') !== -1) {
```

Comment:
This is wrong as it's returning the options without checking if `id_token` is present in the response type. Think of `code id_token`: a`nonce` will be needed anyway.